src/en/developer-guide/applications/utilities/custom-assignment-cascade.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TALXIS Custom Assignment Cascade
+tagline: Enables advanced assignment cascade configurations
+author: František Čapek
+---
+
+# **Assignment Cascade Configuration**
+
+This extension enables custom settings for one-to-many relationship assignment cascade. Use this when you need to make advanced filtering or if you want to use different attribute other than the default _ownerid_. TALXIS Custom Assignment Cascade is part of the Ownership feature.
+
+## Important!
+**You cannot create a configuration for a relationship that has enabled the default assignment cascade in the metadata, change the Assign Cascade to Cascade-None first if you want to use it.**
+
+# Entity
+
+## **TALXIS Configuration - Assignment Cascade**
+| Attribute name 	| Description 	| Type | Requirement
+|-	|-	|- |-
+| talxis_configuration_assignmentcascadeid 	| Unique id.	| Guid | System
+| talxis_filteringattributesavedstate 	| State of filtering attribute - pre-save or post-save.	| Two options | Optional
+| talxis_filterownershipattributename 	| Filtering attribute of type systemuser/team. 	| String | Optional
+| talxis_name 	| Name of the configuration. | String | Required
+| talxis_pluginstepid 	| Id of the associated sdk message processing step. | String (Guid) | Read-only
```

Comment:
I would add "Autofills on save" or something like that so they know they don't have to do anything with this

Reply:
I think it's obvious read-only fields should be left untouched.

Reply:
Sure Im ok with that.